landoui/dockerflow.py

```diff
@@ -16,8 +19,15 @@ def heartbeat():
     and return a 200 iff those services and the app itself are
     performing normally. Return a 5XX if something goes wrong.
     """
-    # TODO check backing services and update tests
-    return '', 200
+    try:
+        response = requests.get(
+            current_app.config['LANDO_API_URL'] + '/__lbheartbeat__'
```

Comment:
Since I saw the same global env var being manipulated 2 different ways in 3 unrelated parts of the code base, I decided to nip the maintenance problem in the bud before it gets worse.  That's why that env var has been copied into the app.config object and isolated to a single app layer.

Environment variables are a kind of global variable, but worse.  By storing our custom data in them we have introduced a 3rd(!) kind of global variable into the application (app.config, python module-level variables, and now env vars).  That's fine when the app is trivial, but when you see multiple different uses of a global variable across the code base like I did, you need to refactor.